Source/Diagnostics/ReducedDiags/RawEFieldReduction.H

```diff
@@ -0,0 +1,284 @@
+/* Copyright 2021 Revathi Jambunathan
```

Comment:
```suggestion
/* Copyright 2021 Revathi Jambunathan, Saurabh Sawant, Andrew Nonaka
```

Reply:
Hahaha, I barely made changes to the file. You wrote it.